src/components/actionBar/index.tsx

```diff
@@ -38,7 +38,7 @@ type Props = {
   disabled?: boolean;
   text?: string | React.ReactNode;
   button?: {
-    text: string;
+    text: Props['btnText'];
```

Comment:
is it really necessary support both btnText and buttonText, can we leave only one?

Reply:
not really, I will refactor 👍